ui/components/app/alert-system/alert-modal/alert-modal.tsx

```diff
@@ -166,7 +166,12 @@ function AlertDetails({
     >
       {customDetails ?? (
         <Box>
-          <Text variant={TextVariant.bodyMd}>{selectedAlert.message}</Text>
+          <Text
+            variant={TextVariant.bodyMd}
```

Comment:
Non-blocking: The `Text` component defaults to `bodyMd`, so specifying it isn't necessary, but since it's already included, it's non-blocking.